docker/Dockerfile

```diff
@@ -0,0 +1,80 @@
+#
+# Dockerfile for Miaou development
+#
+
+
+
+# Based on docker official ubuntu LTS image
+FROM ubuntu:14.04
+
+
+
+# Define Dockerfile maintainer
+MAINTAINER Oliver THEBAULT <contact@oliver-thebault.com>
+
+
+
+# Install Ubuntu miscellaneous packages
+RUN \
+	sed -i 's/# \(.*multiverse$\)/\1/g' /etc/apt/sources.list && \
+	apt-get update && \
+	apt-get -y upgrade && \
+	apt-get install -y \
+		build-essential \
+		software-properties-common \
+		curl wget unzip xz-utils \
+		git nano vim man htop && \
```

Comment:
Why `htop`? Why `git`? I don't think `nano` or `vim` are useful either in the container.